res/menu/upload_files.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:android="http://schemas.android.com/apk/res/android"
+      xmlns:app="http://schemas.android.com/apk/res-auto">
```

Comment:
This should be "app"! See for example here: http://stackoverflow.com/questions/26377305/item-with-appshowasaction-not-showing This is simply due to the fact that we have implemented material design and its action bar via the AppCompat lib not the "native" action bar which wouldn't be material all the way back to version 14. Thus it need to be the re-auto namespace (!)